component/addons/nodeexporter-remove-mountpropagation.libsonnet

```
@@ -0,0 +1,30 @@
+// This addons removes the mountPropagation field from the node-exporter host mounts.
```

```suggestion
// This addon removes the mountPropagation field from the node-exporter host mounts.
```

Would be nice to get an explanation why this addon may be necessary.

---

@TheBigLee What is this used for? I just copied this part around and I'm not sure if it worked before.

Golden says it did not work:

https://github.com/projectsyn/component-syn-kube-prometheus/blob/484df9468ab5d1e6f423f0c88ad60a73188e0d8e/tests/golden/kubernetes_1.21/kubernetes_1.21/kubernetes_1.21/60_node-exporter-daemonset.yaml#L46